### PR TITLE
Preparing PyLint for subpackage breakup.

### PR DESCRIPTION
**Pull request by dhermes**
This change started out in #2367 but I figured we'd get it out of the way here since it's not as much in dispute.

### CODE REVIEW

File: scripts/verify_included_modules.py [hunk 1]
```diff
@@ -172,7 +177,12 @@ def get_parser():
     return parser
 
 
-if __name__ == '__main__':
+def main():
+    """Main script to verify modules included."""
     parser = get_parser()
     args = parser.parse_args()
-    main(build_root=args.build_root)
+    verify_modules(build_root=args.build_root)
```
Comment (dhermes) [inline scripts/verify_included_modules.py]: Note moving the stuff from `__main__` (i.e. global scope) to `main()` was to appease `pylint`. I'm not sure when or why the OS-dependent path `'scripts/verify_included_modules.py'` ended up in `IGNORED_DIRECTORIES` in `run_pylint.py`.

Comment (tseaver) [inline scripts/verify_included_modules.py]: @daspecster added it in 7e1105be.

**Comment by dhermes** [comment]
@tseaver Sorry I just realized I left out the important part: Taking the docstrings out of namespace package `__init__.py`.

**Comment by dhermes** [comment]
@tseaver LMK if the 2nd commit was problematic.

**Comment by tseaver** [comment]
@dhermes Nope.

**Pull request merged**